schematics/nav/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.spec.ts

```diff
@@ -0,0 +1,26 @@
+
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { <%= classify(name) %>Component } from './<%= dasherize(name) %>.component';
+
+describe('<%= classify(name) %>Component', () => {
+  let component: <%= classify(name) %>Component;
+  let fixture: ComponentFixture<<%= classify(name) %>Component>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ <%= classify(name) %>Component ]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
```

Comment:
The two `beforeEach` calls can be combined.

Reply:
This code was copied from the base create component schematic.